pkg/blobinfocache/deprecated.go

```diff
@@ -0,0 +1,35 @@
+package blobinfocache
```

Comment:
So far we’ve not been maintaining API compatibility; is this a good time to start? Is there anyone who would care?

Reply:
I've dropped the deprecated methods with ca54d11 -> 4cd979b.